www/includes/utility.php

```diff
@@ -893,6 +879,8 @@ function entities_to_numbers($string) {
 
 function make_member_url($name, $const = '', $house = HOUSE_TYPE_COMMONS, $pid = NULL) {
 
+    # convert name to ISO so we have sensible URLs
+    $name = iconv('UTF-8', 'ISO-8859-1//TRANSLIT', $name);
```

Comment:
(Or alternatively, we could leave accents alone in the URLs now, we do on other sites. Few enough here, though.)